<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kr4Class</class>
 <widget class="QWidget" name="kr4Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kr4</string>
  </property>
  <widget class="QPushButton" name="Encrip_but">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>РЁРёС„СЂРѕРІР°С‚СЊ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Decrip_but">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Р Р°СЃС€РёС„СЂРѕРІР°С‚СЊ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_but">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Р’С‹С…РѕРґ</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Р’С…РѕРґРЅРѕР№ С„Р°Р№Р»</string>
   </property>
   <widget class="QLineEdit" name="InFileEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Р¤Р°Р№Р» СЃ Р·РІСѓРєРѕРј (СЃС‚РµРіР°РЅРѕРіСЂР°С„РёСЏ)</string>
   </property>
   <widget class="QLineEdit" name="MusFileEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Р’С‹С…РѕРґРЅРѕР№ С„Р°Р№Р»</string>
   </property>
   <widget class="QLineEdit" name="OutFileEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>РљР»СЋС‡ (С€РёС„СЂ Р¦РµР·Р°СЂСЏ/Р’РµР¶РµРЅРµСЂР°)</string>
   </property>
   <widget class="QLineEdit" name="KeyEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>РњРµС‚РѕРґ С€РёС„СЂРѕРІР°РЅРёСЏ</string>
   </property>
   <widget class="QComboBox" name="CheckMode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kr4.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Exit_but</sender>
   <signal>clicked()</signal>
   <receiver>kr4Class</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>

